test/Modal_test.js

```diff
@@ -5,7 +5,7 @@ var TestUtils = require("react-addons-test-utils");
 
 import {Modal} from "../";
 
-describe("Modal", () => {
+describe("Modal", function() {
```

Comment:
`  8:19  warning  Missing function expression name  func-names`

could get around this by moving the function definition outside the call to `describe`. Could also special-case this file by putting a comment at the top: `/*eslint func-names: "off"*/`